docker/scripts/dev_start.sh

```diff
@@ -322,10 +334,6 @@ function main() {
     APOLLO_DEV="apollo_dev_${USER}"
     docker ps -a --format "{{.Names}}" | grep "$APOLLO_DEV" 1>/dev/null
     if [ $? == 0 ]; then
-        if [[ "$(docker inspect --format='{{.Config.Image}}' $APOLLO_DEV 2> /dev/null)" != "$APOLLO_DEV_IMAGE" ]]; then
```

Comment:
No need to remove bazel output here. For CI, we will observe if it runs smoothly.